common/lib/connection_plugin_chain_builder.ts

```diff
@@ -61,6 +62,7 @@ export class ConnectionPluginChainBuilder {
     ["failover", { factory: FailoverPluginFactory, weight: 700 }],
     ["failover2", { factory: Failover2PluginFactory, weight: 710 }],
     ["efm", { factory: HostMonitoringPluginFactory, weight: 800 }],
+    ["efm2", { factory: HostMonitoring2PluginFactory, weight: 810 }],
```

Comment:
Need to add `HostMonitoring2PluginFactory` to the `PLUGIN_WEIGHTS` map below as well. I notice that map is missing some newer plugins.